openreview/conference/templates/decision_process.py

```diff
@@ -0,0 +1,66 @@
+def process_update(client, note, invitation, existing_note):
+
+    CONFERENCE_ID = ''
+    SHORT_PHRASE = ''
+    AUTHORS_NAME = ''
+    AUTHORS_NAME_ACCEPTED = ''
+    EMAIL_AUTHORS = False
+    baseUrl='https://openreview.net'
+
+    forum_note = client.get_note(note.forum)
+   
+    AUTHORS_ID = CONFERENCE_ID + '/Paper' + str(forum_note.number) + '/' + AUTHORS_NAME
+    
+    # Check if the content of the existing note is the same as the updated note
+    if existing_note:
+      equal = True
+      for key in existing_note.content:
+        if note.content[key] and (existing_note.content[key] != note.content[key]):
+          equal = False
+      # If the content is not equal, want to send emails 
+      if(not equal):
+        if (EMAIL_AUTHORS and ('everyone' in note.readers or AUTHORS_ID in note.readers)):
+          groups = [AUTHORS_ID]
+          subject = '{SHORT_PHRASE} Decision posted to your submission - Paper number: {paper_number}, Paper title: {paper_title}'.format(
```

Comment:
You should add square brackets around the venue short name to match how emails are formatted. 

Also, do we want to say that the decision has been updated when there is a change on the content, instead of posted?